index.html

```diff
@@ -0,0 +1,123 @@
+                </div>
+                <p id="amountWanted" class="amountWanted"> </p>
+                <p id="amountResult" class="amountResult"></p>
+                <div id="rates" class="rates"></div>
+            </form>
+
+        </section>
+        <section class="list-currencies">
+            <h1 class="h1-list">List of the currencies</h1>
+            <h1 class="h1-list">Section-2 On Progress</h1>
+
+        </section>
+        <section class="add-update">
+            <h1 class="h1-list">Add or Update Currency</h1>
+            <div class="container">
+                <div class="form-container">
+                    <h2>Add New Currency</h2>
+                    <form id="add-currency-form" onsubmit="addCurrency(event)">
+                        <div class="form-group1">
+                            <label for="new-currency-rate">Rate</label>
+                            <input type="number" step="0.00001" id="new-currency-rate" name="new-currency-rate"
+                                required>
+                        </div>
+                        <div class="form-group1">
+                            <label for="new-currency-code">Currency Code</label>
+                            <input type="text" id="new-currency-code" name="new-currency-code" required>
+                        </div>
+                        <div class="form-group1">
+                            <label for="new-currency-name">Currency Name</label>
+                            <input type="text" id="new-currency-name" name="new-currency-name" required>
+                        </div>
+                        <div class="btn-submit">
+                            <button type="submit" class="button">Add Currency</button>
+                        </div>
+                    </form>
+                </div>
+
+                <div class="form-container">
+                    <h2>Currency Update</h2>
+                    <form id="update-currency-form" onsubmit="updateCurrency(event)">
+                        <div class="form-group1">
+                            <label for="update-rate">Rate</label>
+                            <input type="number" step="0.00001" id="update-rate" name="update-rate" required>
+                        </div>
+                        <div class="form-group1">
+                            <label for="to-currency">Currency</label>
+                            <select class="currency-dropdown" id="to-currency-update" name="to-currency"
+                                required></select>
+                        </div>
+
+                        <div class="btn-submit">
+                            <button type="submit" class="button2">Update Currency</button>
+                        </div>
+                    </form>
+                </div>
+            </div>
+        </section>
+        <section class="most"> This is temporary
+
+        </section>
+        </div>
+    </main>
+</body>
+
```

Comment:
Great handling of events and buttons to activate them. Your skills in managing the DOM are clearly benefiting your users 🤓